sdcm/nemesis.py

```diff
@@ -2324,7 +2322,8 @@ def toggle_table_gc_mode(self):
             new_gc_mode = GcMode.TIMEOUT
         new_gc_mode_as_dict = {'mode': new_gc_mode.value}
 
-        alter_command_prefix = 'ALTER TABLE ' if keyspace_table not in mview_ks_cfs else 'ALTER MATERIALIZED VIEW '
+        alter_command_prefix = 'ALTER TABLE ' if not is_cf_a_view(
```

Comment:
This line is not directly related to the main PR fix, but it was missing on a previous PR and be good to align with it.